perfkitbenchmarker/providers/profitbricks/util.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for working with ProfitBricks resources."""
+
+import logging
+import requests
+
+from perfkitbenchmarker import errors
+from perfkitbenchmarker.providers import profitbricks
+from perfkitbenchmarker.providers.profitbricks import \
+    profitbricks_machine_types
+
+# Global Values
+PROFITBRICKS_API = profitbricks.PROFITBRICKS_API
+FLAVORS = profitbricks_machine_types.FLAVORS
+
+
+def PerformRequest(action, url, header, json=None):
+    """ Makes an HTTP request to the ProfitBricks REST API """
```

Comment:
Would you mind updating all your docstrings to terminate with punctuation (probably just a '.') and additionally strip the spaces at the beginning/end?